agent/service/hostapd.go

```diff
@@ -64,6 +68,9 @@ nas_identifier=%s
 func configHostapd(apConfig *ocstruct.Device, wlanINTFName string) error {
 	hostname := *apConfig.Hostname
 	apRadios := apConfig.Radios
+	ctrlInterface := *apConfig.VendorConfig["ctrl_interface"].ConfigValue
```

Comment:
What if there is no "ctrl_interface" config, for the case that it is not running with Faucet. I thing this one will throw NullpointerException. Same for the next line.